### PR TITLE
Fix jpegxr

### LINKED ISSUE #62: Can't enable `jpegxr` feature
The `jpegxr` feature (disabled by default in Ruffle) uses C/C++ code, and the build system doesn't find the cross-compiler for it correctly.

### PR DESCRIPTION
**Pull request by Fancy2209**
One flag was all that was needed.
Fixes #62

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -33,6 +33,7 @@ ruffle_core = { git = "https://github.com/ruffle-rs/ruffle.git", branch = "maste
     "lzma",
     "default_compatibility_rules",
     "default_font",
+    "jpegxr"
```
Comment (Dinnerbone) [inline Cargo.toml:36]: We should enable it in the build script, or this forces every developer to figure out how to set up jpegxr too

Comment (Fancy2209) [reply Cargo.toml]: Ah good point! Should be fixed now

**Comment by Fancy2209** [comment]
Now it should be right, sorry about that. 😅

**Comment by torokati44** [comment]
Are there any SWFs that I could use for testing this? 
I tried for one among the Ruffle test cases, but haven't found one; and also in the issues linked to https://github.com/ruffle-rs/ruffle/pull/13180, but those didn't seem to work (possible script execution timeout).

**Comment by sleepycatcoding** [comment]
> I tried for one among the Ruffle test cases, but haven't found one;

Uhh, aren't `stage3d_texture_bytearray_*` tests supposed to cover this?

**Comment by torokati44** [comment]
> > I tried for one among the Ruffle test cases, but haven't found one;

> Uhh, aren't the stage3d_texture_bytearray_* tests supposed to cover this? The TOML files for these tests explicitly require the jpegxr feature.

Well I didn't say there aren't any, just that I haven't found them... :P
And apparently, yep, at least the one I checked first does indeed cover it. Thank you!

**Review by torokati44 — APPROVED** [review]
Yep, it works, thank you so much! ^^

**Comment by torokati44** [comment]
So sorry for the fumblationing that was merging this unsquashed... 😞
It still won't let me be.

**Comment by Fancy2209** [comment]
I feel kinda guilty too 
I thought about squashing it manually but never did

**Comment by torokati44** [comment]
Oh, don't! You're not at fault. It's no big deal, I just have to get these little annoyances out of my system... :no_mouth:

**Pull request merged**